src/dataflow-types/src/client.rs

```diff
@@ -141,7 +140,13 @@ pub enum StorageCommand<T = mz_repr::Timestamp> {
     /// Create the enumerated sources, each associated with its identifier.
     ///
     /// For each identifier, there is a source description and a valid `since` frontier.
```

Comment:
Would you be willing to update the doc comment to explain the additional vector?